index.html

```diff
@@ -6,66 +6,64 @@
-                <h1 class="article__title">THE TARGET</h1>
+                <h1 class="article__title">the target</h1>
                 <p class="article__content">Morbi non efficitur nibh. duis amet eroest Fusce faucibus ante libero ital luctus egestas sed.</p>
             </div>
             <div class="article">
                 <span class="article__image article__image_circle article__image_icon_time"></span>
-                <h1 class="article__title">THE TIME</h1>
+                <h1 class="article__title">the time</h1>
                 <p class="article__content">Morbi non efficitur nibh. duis amet eroest Fusce faucibus ante libero ital luctus egestas sed.</p>
             </div>
         </div>
         
         <div class="article">
             <img class="article__image article__image_big">
-            <p class="article__content article__content_big">DUIS EGET EUISMOD ET LOREM PLACERAT TORTOR PROIN CONSEQUAT QUAM EGET CONVALLIS ELEIFEND, LIBERO JUSTO.</p>
+            <p class="article__content article__content_big">duis eget euismod et lorem placerat tortor proin concequat quam eget convallis eleifend, libero justo.</p>
             <div class="article__border"></div>
             <p class="article__author">John Doe</p>
         </div>
         
         <div class="content-section">
-            <h1 class="content-section__title"></h1>
+            <h1 class="content-section__title">the blog</h1>
             <div class="content-section__border"></div>
             <div class="article">
                 <img class="article__image article__image_square">
-                <h1 class="article__title">FUSTCE FAUCIBUSY ANTE LIBERO</h1>
+                <h1 class="article__title">fuctce faucibusy ante libero</h1>
                 <p class="article__content">Cras interdum vestum a dolor amet tursisi ligula rhoncus sodales...</p>
             </div>
             <div class="article">
                 <img class="article__image article__image_square">
-                <h1 class="article__title">FUSTCE FAUCIBUSY ANTE LIBERO</h1>
+                <h1 class="article__title">fustce faucibusy ante libero</h1>
                 <p class="article__content">Cras interdum vestum a dolor amet tursisi ligula rhoncus sodales...</p>
             </div>
             <div class="article">
                 <img class="article__image article__image_square">
-                <h1 class="article__title">FUSTCE FAUCIBUSY ANTE LIBERO</h1>
+                <h1 class="article__title">fustce faucibusy ante libero</h1>
                 <p class="article__content">Cras interdum vestum a dolor amet tursisi ligula rhoncus sodales...</p>
             </div>
         </div>
         
         <div class="content-section">
-            <h1 class="content-section__title"></h1>
+            <h1 class="content-section__title">the gallery</h1>
             <div class="content-section__border"></div>
             <div class="gallery">
```

Comment:
I think that we can easy use one block, with 4 elements inside. Can we?

Reply:
Should I create modificators for "gallery__link" elements to split gallery in two lines?

Reply:
No